src/NuGet.Clients/NuGet.CommandLine/Common/Console.cs

```diff
@@ -252,23 +263,30 @@ public void PrintJustified(int startIndex, string text, int maxWidth)
         {
             if (maxWidth > startIndex)
             {
-                maxWidth = maxWidth - startIndex - 1;
+                maxWidth = maxWidth - startIndex;
             }
 
             lock (_writerLock)
             {
                 while (text.Length > 0)
                 {
                     // Trim whitespace at the beginning
-                    text = text.TrimStart();
+                    text = text.TrimStart(' ', '\t');
                     // Calculate the number of chars to print based on the width of the System.Console
                     int length = Math.Min(text.Length, maxWidth);
 
                     string content;
 
-                    // Text we can print without overflowing the System.Console, excluding new line characters.
-                    int newLineIndex = text.IndexOf(Environment.NewLine, 0, length, StringComparison.OrdinalIgnoreCase);
-                    if (newLineIndex > -1)
+                    // Get the index of the newLine character upto 'length' characters
+                    (int newLineIndex, bool rn) = NewLineIndex(text, length);
```

Comment:
`newLine`?

Reply:
`rn = '\r\n'`. So perhaps a better name for it would be `windowsNewLine`?

Reply:
As a variable name, that's still not very clear, and I prefer the change to `newLine`.
Actually, I realize some on the team will complain that it doesn't have the `is` prefix. I'm not a stickler for that, but since it's a bool, should actually be `isNewLine`. Maybe wait until someone else chimes in

Reply:
I vote for `isNewLine`

>  DO NOT use any acronyms that are not widely accepted, and even if they are, only when necessary.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions#using-abbreviations-and-acronyms

>  DO name Boolean properties with an affirmative phrase (CanSeek instead of CantSeek). Optionally, you can also prefix Boolean properties with "Is", "Can", or "Has", but only where it adds value.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members?redirectedfrom=MSDN#names-of-properties

Reply:
Answer this first, because this method may go away https://github.com/NuGet/NuGet.Client/pull/3489/files#r449243866

Reply:
maybe `isCRLF` ;)

How's that, @donnie-msft? The point is to know if it's a two-part newline or a single-part newline.